CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * [CHANGE] grpcutil.Update: Remove gRPC LB related metadata. #102
 * [CHANGE] concurrency.ForEach: deprecated and reimplemented by new `concurrency.ForEachJob`. #113
 * [CHANGE] ring/client: It's now possible to set different value than `consul` as default KV store. #120
+* [CHANGE] Lifecycler: Change default value of `-[ingester.]final-sleep` (YAML: `[ingester.]final_sleep`) to `0s`.  #121
```

Comment:
Let's not use ambiguous command line options here, we can just say "lifecycler's final-sleep" directly.

Reply:
cleaned it up a bit in https://github.com/grafana/dskit/pull/121/commits/66acab096ac4ccdb1523a9e9d6175e2e29516130